Tests/LSPLoggingTests/SupportTests.swift

```diff
@@ -23,10 +23,10 @@ final class SupportTests: XCTestCase {
       messages.append((message, level))
     }
 
-    func check(expected: [(String, LogLevel)], file: StaticString = #file, line: UInt = #line) {
+    func check(expected: [(String, LogLevel)], file: StaticString = #filePath, line: UInt = #line) {
       testLogger.flush()
-      XCTAssert(messages.count == expected.count, "\(messages) does not match expected \(expected)", file: file, line: line)
-      XCTAssert(zip(messages, expected).allSatisfy({ $0.0 == $0.1 }), "\(messages) does not match expected \(expected)", file: file, line: line)
+      XCTAssert(messages.count == expected.count, "\(messages) does not match expected \(expected)", file: (file), line: line)
```

Comment:
I see you updated some of the (file) cases, but it seems there are still a few more (you can search the patch for "(file)" to find the rest).

Reply:
yep - I apologize for not being explicit, I hit the easy stuff, but I want to read and understand what's being done with some of the functions, and just haven't had the time to dig into it when I wasn't panicking about some external thing or another. I'm still working on it, but I don't yet know this codebase as tightly as I probably should to make these updates, so I'm learning and digging to I don't regress what's happening.